web/pages/jobs.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState, useEffect } from 'react';
+import { Box, Heading } from '@chakra-ui/react';
+import Posts from '../components/Posts';
+import Pagination from '../components/Pagination';
+
+const App = () => {
+	const [posts, setPosts] = useState([]);
+	const [loading, setLoading] = useState(false);
+	const [currentPage, setCurrentPage] = useState(1);
+	const [postsPerPage] = useState(5);
+
+	useEffect(() => {
+		const fetchPosts = async () => {
+			setLoading(true);
+			const res = await fetch('http://localhost:3000/api/jobs').then((response) =>
```

Comment:
should be `fetch('/api/jobs')`
Also if we're awaiting a `res` then why are we also using the promise? Probably remove the .then()